scripts/lib/libvirt/templates/admin-node.xml

```diff
@@ -5,8 +5,8 @@
   <vcpu>$adminvcpus</vcpu>
   <os>
     <type arch='$march' machine='$machine'>hvm</type>
-$osloader
-    <nvram>/var/lib/libvirt/$cloud-admin_VARS.fd</nvram>
+    $osloader
```

Comment:
this was intentionally not properly indented in order to not produce trailing whitespace in case $osloader expands to empty.